k8s/deployment.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: house-innovations-housnav-web-deployment
+  labels:
+    app: house-innovations-housnav-web
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: house-innovations-housnav-web
+  template:
+    metadata:
+      labels:
+        app: house-innovations-housnav-web
+    spec:
+      serviceAccountName: github-actions
+      containers:
+        - name: house-innovations-housnav-web
+          image: ${OPENSHIFT_REGISTRY_REPOSITORY}:${GIT_SHA}
+          ports:
+            - containerPort: 3000
+          livenessProbe:
+            httpGet:
+              path: /
+              port: 3000
+            initialDelaySeconds: 30
+            periodSeconds: 10
+          readinessProbe:
+            httpGet:
+              path: /
+              port: 3000
+            initialDelaySeconds: 30
+            periodSeconds: 10
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+  revisionHistoryLimit: 3
+  rollbackTo:
+    revision: 1
+---
```

Comment:
What are these --- for?

Reply:
It's syntax for separating the deployment sections.